AirCasting/SessionViews/EditView.swift

```diff
@@ -0,0 +1,100 @@
+// Created by Lunar on 08/07/2021.
+//
+
+import SwiftUI
+import AirCastingStyling
+
+struct EditView: View {
+    
+    @State private var isSessionDownloaded = false
+    private let measurementStreamStorage: MeasurementStreamStorage
+    let sessionUUID: SessionUUID
+    private let sessionSynchronizer: SessionSynchronizer
+    let sessionUpdateService: SessionUpdateService
+    @Binding var showModalEdit: Bool
+    @StateObject private var editSessionViewModel: EditSessionViewModel
+
+    
+    init(measurementStreamStorage: MeasurementStreamStorage, sessionUUID: SessionUUID, sessionSynchronizer: SessionSynchronizer, sessionUpdateService: SessionUpdateService, showModalEdit: Binding<Bool>) {
```

Comment:
why aren't we passing the viewmodel itself here? We want to hide this dependencies from views and make it only know about its viewmodel.

Since we don't have any DI system in place this is not *that* important, but overall this is the practice to follow. We should also make VMs protocols to break dependencies. Reason is:
* If the view depends on a concrete (non-protocol) VM it will have to recompile anytime anything in VM changes - even an implementation of a completely private VM method.
* If anything that this VM depends on it will also have to recompile, so the view will also recompile
* If we turn this View-VM dependency to a protocol it will only have to recompile when this protocol changes, so any implementation change to VM that doesn't affect it's public interface (the protocol) won't trigger any recompilation of the view. This is basically the whole idea behind polymorphism and abstract interfaces - Dependency Inversion.